client/packages/system/src/ContactTrace/DetailView/DetailView.tsx

```diff
@@ -38,7 +38,7 @@ export const DetailView: FC<DetailViewProps> = ({
 }) => {
   const t = useTranslation('dispensary');
   const navigate = useNavigate();
-  const { setSuffix, urlParts } = useBreadcrumbs([AppRoute.ContactTrace]);
```

Comment:
Only need to pass in `topLevelRoutes` ... like for the Sync and Settings pages, not needed here